main/src/com/google/refine/io/FileProjectManager.java

```diff
@@ -377,6 +378,13 @@ protected boolean loadFromFile(File file) {
         if (file.exists() || file.canRead()) {
 	        try {
 	        	ParsingUtilities.mapper.readerForUpdating(this).readValue(file);
+
+                PreferenceStore ps = this.getPreferenceStore();
+                if (ps.get("userLang") != null) {
+                    Locale locale = new Locale.Builder().setLanguage((String) ps.get("userLang")).build();
```

Comment:
Do you know if the language codes we use actually match those that are supported by `java.util.Locale`? I know there are subtle differences between various language identifier schemes, it would be good to double-check that Weblate and java.util.Locale agree.

Reply:
I did not check for this, but I will. Thanks very much.

Reply:
I did notice some differences in the codes for Japanese, Traditional/Simplified Chinese. Made some updates addressing this.